static/files/forms-data.json

```diff
@@ -404,6 +404,61 @@
           ]
         }
       ]
+    },
+    "/solutions/open-source-security/cyber-resilience-act": {
+      "templatePath": "/solutions/open-source-security/cyber-resilience-act/index.html",
+      "isModal": true,
+      "modalId": "cra-contact-modal",
+      "formData": {
+        "title": "Get in touch with our experts on the Cyber Resilience Act (CRA) compliance",
+        "formId": "6051",
+        "returnUrl": "https://canonical.com/solutions/open-source-security/cyber-resilience-act/index.html#contact-form-success",
```

Comment:
Not sure why, but in the demo I am still being redirected to u.com

Reply:
Works correct locally. Not sure why it doesn't work in the demo.

Reply:
Works correctly in the demo now. Might have been a demo build error.